Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -357,11 +357,15 @@ logOutPopUpModelConfig state =
           optionButtonOrientation = "VERTICAL"
           , backgroundClickable = true
           , customerTipAvailable = true
+          , fareEstimateText =  getString FARE_ESTIMATE
+          , tipSelectedText = getString TIP_SELECTED
+          , fareEstimate =  if state.data.rateCard.additionalFare == 0 then "₹" <> (show state.data.suggestedAmount) else  "₹" <> (show state.data.suggestedAmount) <> " - " <> "₹" <> (show $ state.data.suggestedAmount + state.data.rateCard.additionalFare)
+          , tipSelected = if (state.props.customerTip.tipActiveIndex == 0) then "-" else (" ₹"<> (fromMaybe "" (["0", "10", "20", "30"] DA.!! state.props.customerTip.tipActiveIndex)))
           , dismissPopup = true
           , customerTipArray = [(getString NO_TIP), "₹10 🙂", "₹20 😄", "₹30 🤩"]
           , customerTipArrayWithValues = [0,10, 20, 30]
           , primaryText {
-              text =  if(isLocalStageOn ST.QuoteList)then (getString TRY_AGAIN_WITH_A_TIP) else (getString SEARCH_AGAIN_WITH_A_TIP)
+              text =  if(isLocalStageOn ST.QuoteList)then ((getString TRY_AGAIN) <> "?") else (getString SEARCH_AGAIN_WITH_A_TIP)
```

Comment:
Can we written as :
text = if isLocalStageOn ST.QuoteList then (getString TRY_AGAIN <> "?") else getString SEARCH_AGAIN_WITH_A_TIP

Reply:
done